source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/observations.py

```diff
@@ -22,6 +22,8 @@
 if TYPE_CHECKING:
     from omni.isaac.lab.envs import ManagerBasedEnv, ManagerBasedRLEnv
 
+from torchvision import models
```

Comment:
Please modify the `pyproject.toml` file to have this as thirdparty dependencies (so it moves up with other standard python imports)

Reply:
How do you feel about adding Transformers as an extra standard library @Mayankm96 ? I have code that also adds Theia (https://theia.theaiinstitute.com/) as an available model, but it depends on transformers. In my tests, the step efficiency of Theia blew ResNet18 out of the water.

I believe they also have a permissive license https://github.com/huggingface/transformers/blob/main/LICENSE . 

If you are cool with adding transformers, I can put that in

Reply:
I think having the Transformers library would be a great addition.